src/accordion.js

```diff
@@ -39,7 +45,7 @@ Accordion.prototype.findTriggers = function() {
 };
 
 Accordion.prototype.setAria = function(trigger, index) {
-  var contentID = 'content-' + index;
+  var contentID = this.opts.contentPrefix + '-' + 'content-' + index;
```

Comment:
Do we want the accordion to set IDs for all the trigger elements? What about letting users set button IDs in their markup instead? Then the `setAria` method can check for an ID, and if it's present, update ARIA attributes. That way, we wouldn't be changing IDs of existing elements, which I think would be a good thing.